### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -49,33 +49,33 @@
...
     "@fiaRegulations": {},
     "freePracticeFirstLetter": "FP",
     "@freePracticeFirstLetter": {},
-    "freePracticeOne": "Free Practice 1",
+    "freePracticeOne": "Free Practice FP1",
     "@freePracticeOne": {},
-    "freePracticeTwo": "Free Practice 2",
+    "freePracticeTwo": "Free Practice FP2",
     "@freePracticeTwo": {},
-    "freePracticeThree": "Free Practice 3",
```
Comment (BrightDV) [inline lib/l10n/app_en.arb:64]: Better to keep it to Free Practice FP3

**Review by BrightDV — CHANGES_REQUESTED** [review]
Thanks for the PR! Only a small request from my side.

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -49,33 +49,33 @@
...
+    "freePracticeThree": "Final practice FP3",
```
[the remark this answers is not in the document]
Comment (sinfullad) [reply lib/l10n/app_en.arb:64]: the Prix' of the Grand Prix' entered confused me a bit too

we should just follow how F1 says it as Grands Prix entered

**Comment by BrightDV** [comment]
I finally made a commit on my side with my suggestions and added you as a co-author.
Thanks for PR!

**Pull request closed without merging**